src/ImageSharp/Formats/Ico/Common/IcoLikeFrameCompression.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+namespace SixLabors.ImageSharp.Formats.Ico.Common;
+
+/// <summary>
+/// IcoFrameCompression
+/// </summary>
+public enum IcoLikeFrameCompression
```

Comment:
Use separate types. That they use the same compression approach is an implementation detail.